### PR TITLE
Fix/docs update

### PR DESCRIPTION
**Pull request by rb80**
Updated markdown formatting for style

### CODE REVIEW

File: docs/Instructions.md [hunk 1]
```diff
@@ -44,22 +44,20 @@ To install this on a Bedrock server takes a little more work, but roughly:
     development_behavior_packs\logrief\pack_icon.png
     development_behavior_packs\logrief\scripts
     development_behavior_packs\logrief\scripts\main.js
-4) Run your server and after it has started shut it down.
+4) Run your server and after it has started shut//////////////////////////////// it down.
```
Comment (CorrodedCoder) [inline docs/Instructions.md]: Did you fall asleep on the keyboard? 😆

**Review by CorrodedCoder — CHANGES_REQUESTED** [review]
Everything looks great except the mysterious ///////////////////////////// in the instructions.

File: docs/Instructions.md (hunk 1 above)
Comment (rb80) [reply docs/Instructions.md]: That was Rio's contribution!

**Comment by rb80** [comment]
D'oh that was cat typing!

**Review by CorrodedCoder — APPROVED** [review]

**Pull request merged**